File: 3.compareTheTripletProblem.js
function compareTriplets(a, b) {
    let outputArray = [0, 0];
    const numberOfQuestions = 3;
    
    for (let i = 0; i < numberOfQuestions; i++) {
        if(a[i] > b[i]) {
            outputArray[0] += 1;
        }else if(a[i] < b[i]) {
            outputArray[1] += 1;
        }else {
            continue
        }
    };
    
    return outputArray;

};

// console.log(compareTriplets([10, 11, 48], [20, 11, 47]));

File: 3.compareTheTripletProblem.py
def compare_triplets(a, b):
    output_array = [0, 0]
    number_of_questions = 3

    for index in range(number_of_questions):
        if a[index] > b[index]:
            output_array[0] += 1
        elif a[index] < b[index]:
            output_array[1] += 1
        else:
            continue
    
    return output_array


print(compare_triplets([10, 11, 48], [20, 11, 47]))